### PR TITLE
Refractor the Docs, and enables many features.

### PR DESCRIPTION
**Pull request by FarisZR**
## This PR refractor's the Docs, and enables many features.
I have rewritten some parts of the index page, and removed the change log page as this should be done on git.
other features:
- Move all documentation files to the docs file
- use torjoman.aosus.dev domain
- Add GitHub repo and edit URI
- Follow Aosus Color scheme
- Enable auto dark mode
- Use Almarai font
- Create GitHub Actions files for build and PR test
- Add requirements.txt file for PIP
- Enable Aosus Plausible analytics
- Enable MkDocs minify
- Enable Git revesion date
- Enable instant navigation
- Enable auto links

### CODE REVIEW

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+site_name:  Torjoman | تُرجمان
+docs_dir: 'source'
+site_url: https://torojoman.aosus.dev
+repo_name: aosus/Torjoman
+repo_url: https://github.com/aosus/Torjoman
+edit_uri: https://github.com/aosus/Torjoman/_edit/dev/docs/source
```
Comment (TheAwiteb) [inline docs/mkdocs.yml]: الرابط يرجع 404

File: docs/source/index.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# مرحبا بكم في توثيق تُرجمان
+
+المشروع مازال في مرحلة التخطيط
+
+# نبذة عن المشروع
+
+تُرجمان يهدف إلى تحسين الترجمة والمشاركة فيها بطريقة تفاعلية مع المستخدمين عبر تسهيل الترجمة لتكون متوفرة على منصاتهم المفضلة, وجعلها اسرع واكثر متعة.
```
Comment (TheAwiteb) [inline docs/source/index.md]: استخدام الفاصلة `،`

File: docs/source/SRS.md [hunk 3]
```diff
@@ -73,16 +71,16 @@
...
 ٤- حفظ عدد الترجمات لكل كلمة 
 
-٥- التعرف على هوية المترجم من المعرف القادم على الواجهة
+٥- التعرف على هوية المترجم عبر المعرف القادم من الواجهه
 
-٦- إمكانية تقديم برمجيات بديلة للترجمة المقدمة من المترجمين 
+٦- إمكانية تقديم البرمجية ترجمات بديلة وهي الترجمات الاقل شهره من المترجمين 
 
-٧- إمكانية تغير الترجمة  بناءا على أشهر ترجمة   
+٧- اختيار اشهر ترجمة بشكل تلقائي, مع وجود امكانيه اختيار ترجمات الاقل شهره
```
Comment (TheAwiteb) [inline docs/source/SRS.md]: استخدام الفاصلة `،`

**Review by TheAwiteb — CHANGES_REQUESTED** [review]

File: docs/mkdocs.yml (hunk 1 above)
Comment (FarisZR) [reply docs/mkdocs.yml]: رابط التعديل يستخدم مع اسم الملف.

**Comment by TheAwiteb** [comment]
استبدل الفاصلة من `,` الى `،`

File: docs/mkdocs.yml (hunk 1 above)
Comment (FarisZR) [reply docs/mkdocs.yml]: Edit links has been updated and fixed

**Review by TheAwiteb — APPROVED** [review]
LGTM

**Pull request merged**